packages/data-store-common/src/entities/connection/PartyEntity.ts

```diff
@@ -9,14 +10,22 @@ export class PartyEntity extends BaseEntity {
   @Column({ length: 255, nullable: false, unique: true })
   name!: string
 
+  @Column({ length: 255, nullable: true, unique: true })
+  alias!: string
+
+  @Column({ length: 255, nullable: true, unique: false })
```

Comment:
Seems like it should not be null. And following line has assertion to ensure it to be mandatory. 
So nullable should be false for this.

Reply:
incorrect. this can be null

Reply:
to include an uri field. field should be mandatory as a contact should always have an uri

This is a quote from the ticket. Ticket is wrong ? @BtencateSphereon

Reply:
@BtencateSphereon any response? Is the ticket wrong or the code ?